dot/config.go

```diff
@@ -83,13 +83,14 @@ type NetworkConfig struct {
 
 // CoreConfig is to marshal/unmarshal toml core config vars
 type CoreConfig struct {
-	Roles            byte
-	BabeAuthority    bool
-	GrandpaAuthority bool
-	BabeThreshold    *big.Int
-	SlotDuration     uint64
-	EpochLength      uint64
-	WasmInterpreter  string
+	Roles                    byte
+	BabeAuthority            bool
+	GrandpaAuthority         bool
+	BabeThresholdNumerator   *big.Int
+	BabeThresholdDenominator *big.Int
```

Comment:
are `*big.Int`s able to be marshaled to/from toml?

Reply:
I'm not sure, I haven't set up marshaling just yet